### PR TITLE
Hero

### PR DESCRIPTION
**Pull request by mamatkulov95**
Hero section. 
image

### CODE REVIEW

**Comment by mamatkulov95** [comment]
Good evening.
This section is ready to review!
@kumbu132 @abanseka

File: next.config.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
+  // basePath: "./src",
```
Comment (kumbu132) [inline next.config.js]: - [x] Don't commit commented code to master.

File: tailwind.config.js [hunk 2]
```diff
@@ -6,8 +6,30 @@ module.exports = {
...
+
+      lg: "1024px",
+      // => @media (min-width: 1024px) { ... }
+
+      xl: "1280px",
+      // => @media (min-width: 1280px) { ... }
+
+      "2xl": "1536px",
+      // => @media (min-width: 1536px) { ... }
+    },
```
Comment (kumbu132) [inline tailwind.config.js]: [ ] Don't commit commented code to master

**Review by kumbu132 — CHANGES_REQUESTED** [review]
Don't have both a `package-lock.json` and a `yarn.lock` file. Decide to either use `npm` or `yarn` but not both. I would recommend using yarn where possible.

**Comment by kumbu132** [comment]
you have conflicts

File: next.config.js (hunk 1 above)
Comment (mamatkulov95) [reply next.config.js]: changed.

File: tailwind.config.js (hunk 2 above)
Comment (mamatkulov95) [reply tailwind.config.js]: Deleted commented code .

**Comment by mamatkulov95** [comment]
Thank you bro. I will resolve soon.

File: .gitignore [hunk 3]
```diff
@@ -6,7 +6,7 @@
 .pnp.js
 package-lock.json
 next-env.d.ts
-
```
Comment (kumbu132) [inline .gitignore]: lock files should be committed ntotignored

**Review by kumbu132 — APPROVED** [review]

**Pull request merged**